rubicon/objc/runtime.py

```diff
@@ -2096,20 +2110,43 @@ def __init__(self, func, restype=NOTHING, *arg_types):
         self.cfunc_type = CFUNCTYPE(restype, c_void_p, *signature)
 
         self.literal = BlockLiteral()
-        self.literal.isa = addressof(_NSConcreteGlobalBlock)
-        self.literal.flags = BlockConsts.HAS_STRET | BlockConsts.HAS_SIGNATURE
+        self.literal.isa = addressof(_NSConcreteStackBlock)
+        self.literal.flags = BlockConsts.HAS_STRET | BlockConsts.HAS_SIGNATURE | BlockConsts.HAS_COPY_DISPOSE
         self.literal.reserved = 0
-        self.cfunc = self.cfunc_type(self.wrapper)
-        self.literal.invoke = cast(self.cfunc, c_void_p)
+        self.cfunc_wrapper = self.cfunc_type(self.wrapper)
+        self.literal.invoke = cast(self.cfunc_wrapper, c_void_p)
         self.descriptor = BlockDescriptor()
         self.descriptor.reserved = 0
         self.descriptor.size = sizeof(BlockLiteral)
 
+        self.cfunc_copy_helper = _cfunc_type_block_copy(self.copy_helper)
+        self.cfunc_dispose_helper = _cfunc_type_block_dispose(self.dispose_helper)
+        self.descriptor.copy_helper = self.cfunc_copy_helper
+        self.descriptor.dispose_helper = self.cfunc_dispose_helper
+
         self.descriptor.signature = encoding_for_ctype(restype) + b'@?' + b''.join(
             encoding_for_ctype(arg) for arg in signature
         )
         self.literal.descriptor = cast(byref(self.descriptor), c_void_p)
         self.block = cast(byref(self.literal), objc_block)
 
+    @property
+    def _as_parameter_(self):
+        return self.block
```

Comment:
If the value for `_as_parameter_` is always the same for each instance, you can also use a regular attribute instad of a property. Doing `self._as_parameter_ = self.block` in `__init__` should work - that's how it's done [in `ObjCInstance`](https://github.com/pybee/rubicon-objc/blob/master/rubicon/objc/runtime.py#L1284) as well.

There are situations where a property is better than an attribute, for example if `block` was meant to be changeable by the user. But since that isn't the case here, I think an attribute is better, because it's shorter and consistent with our other code.

Reply:
Ok, done.  Old C habit I guess that I did it that way -- sort of defensive programming was the rationale.  But yeah, if it's consistent with how it's already done, sure.  Changed.  Will do your other suggestion shortly too.